File: C#/백준/Bronze/26933. Receptet/Receptet.cs
using System;

namespace q{
    class w{
        static void Main(){
            int t = int.Parse(Console.ReadLine());
            int s = 0;
            for (int i=0; i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                if(aa[0]<aa[1]){
                    s+=(aa[1]-aa[0])*aa[2];
                }
            }
            Console.WriteLine(s);
        }
    }
}

File: Python/백준/Bronze/26933. Receptet/Receptet.py
t = int(input())
s = 0 #필요한 돈
for i in range(t):
    a,b,c = map(int,input().split()) #가진 재료 수, 필요 재료 수, 재료비(1개당)
    if(a>=b):
        continue
    else:
        s+=(b-a)*c
print(s)